feature/experimental/p4rt/internal/p4rtutils/p4rtutils.go

```diff
@@ -182,7 +188,40 @@ func ACLWbbIngressTableEntryGet(infoList []*ACLWbbIngressTableEntryInfo) []*p4_v
 	return updates
 }
 
-func explicitP4RTNodes() map[string]string {
+func explicitP4RTNodes(t *testing.T, dut *ondatra.DUTDevice) map[string]string {
+	t.Helper()
+	if dut.Vendor() == ondatra.CISCO {
```

Comment:
Let's factor out the Cisco specific portion to its own function and leave the original function true to its intended purpose.

```go
// inferP4RTNodesCisco infers the P4RT node name from the port name and device model
// for Cisco devices.
func inferP4RTNodesCisco(...) map[string]string {
  ...
}
```

This should be called from P4RTNodesByPort() below.

Reply:
Thanks for the suggestion. Refactored.